tests/clusterclient.c

```diff
@@ -89,8 +89,8 @@ int main(int argc, char **argv) {
     }
 
     if (redisClusterConnect2(cc) != REDIS_OK) {
-        fprintf(stderr, "Connect error: %s\n", cc->errstr);
-        exit(2);
+        printf("Connect error: %s\n", cc->errstr);
+        exit(1);
```

Comment:
1?

Shouldn't we have the same error codes in the same situations for clusterclient and clusterclient_async?

Btw, It'd still be nice to have the exit codes listed in this comment:

```
/*
 * This program connects to a cluster and then reads commands from stdin, such
 * as "SET foo bar", one per line and prints the results to stdout.
 *
 * The behaviour of the client can be altered by following action commands:
 *
 * !all    - Send each command to all nodes in the cluster.
 *           Will send following commands using the `..ToNode()` API and a
 *           cluster node iterator to send each command to all known nodes.
 *
 */
```

Reply:
Added comment on top and reverted change.